tokens.html

```diff
@@ -0,0 +1,313 @@
+		<pre><code class="language-javascript">x = y
+x *= y
+x !== y
+x++
+-y
+x &gt;&gt;&gt; y
+x || y
+z ? x : y</code></pre></dd>
+
+		<dt><code>variable</code></dt>
+		<dd>An object that is assigned and holds a specific value.
+		<pre><code class="language-less">@nice-blue: #5B83AD;
+@light-blue: lighten(@nice-blue, 20%);</code></pre></dd>
+
+		<dt><code>constant</code></dt>
+		<dd>The name of a a variable that cannot be modified later on.
+		<pre><code class="language-javascript">const PI = 3.14159;</code></pre></dd>
+
+		<dt><code>property</code></dt>
+		<dd>An attribute/characteristic.
+		<pre><code class="language-css">body {
+	color: red;
+	line-height: normal;
+}</code></pre>
+		<pre><code class="language-json">{
+	"data": {
+		"labels": [
+			"foo",
+			"bar"
+		],
+		"series": [
+			[ 0, 1, 2, 3 ],
+			[ 0, -4, -8, -12 ]
+		]
+	},
+	// we even support comments
+	"error": null,
+	"status": "Ok"
+}</code></pre></dd>
+
+		<dt><code>punctuation</code></dt>
+		<dd>Punctuation that is not contained within a string, such as brackets.
+		<pre><code class="language-python">def median(pool):
+	'''Statistical median to demonstrate doctest.
+	>>> median([2, 9, 9, 7, 9, 2, 4, 5, 8])
+	7
+	'''
+	copy = sorted(pool)
+	size = len(copy)
+	if size % 2 == 1:
+		return copy[(size - 1) / 2]
+	else:
+		return (copy[size/2 - 1] + copy[size/2]) / 2</code></pre></dd>
+
+		<dt><code>important</code></dt>
+		<dd>Anything that is important.
+		<pre><code class="language-css">body {
+	color: red !important;
+}</code></pre>
+		<pre><code class="language-markdown"># This is a heading. Headings are important.</code></pre></dd>
+
+		<dt><code>comment</code></dt>
+		<dd>Code comments.
+		<pre><code class="language-markup">&lt;!-- Here's a comment --&gt;
+&lt;style&gt;
+	/* Here's another comment */
+&lt;/style&gt;
+&lt;script&gt;
+// Here's yet another comment
+&lt;/script&gt;</code></pre></dd>
+
+		<dt><code>inserted</code></dt>
+		<dd>Added or modified line, mainly for diffs. In general, also the idea of something being increased.
+		<pre><code class="language-diff">+		// Calculate the area of the circle
+		let area = PI * (radius ** 2);</code></pre></dd>
+
+		<dt><code>deleted</code></dt>
+		<dd>Deleted line, mainly for diffs. In general, also the idea of something being decreased/removed.
+		<pre><code class="language-diff">-		// TODO: Remove this comment
+		let foo = 'bar';</code></pre></dd>
+
+		<dt><code>bold</code></dt>
+		<dd>Bolded text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">**I am bolded text!**</code></pre></dd>
+
+		<dt><code>italic</code></dt>
+		<dd>Italicised text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">*I am italicised text!*</code></pre></dd>
+
+		<dt><code>tag</code></dt>
+		<dd>The markup version of a keyword, to indicate when an element begins and ends.
```

Comment:
A tag is quite different from a keyword. It's its own concept, so I think a short generic description is most fitting.

```suggestion
		<dd>A markup tag (e.g. HTML and XML tags).
```